### DISCUSSION TITLE
FSX

### DISCUSSION THREAD

**Discussion by Andybrwn7** [nguyenquyhy/Flight-Tracker-StreamDeck#91] [General]
I think the program is great, I have set up a few planes on MSFS20. I was wondering if anybody has converted it to work with FSX. I know it's old. But I have so many aircraft in there I like flying. Any help would be appreciated.

**Comment by Crunchmeister1** [comment]
> I was wondering if anybody has converted it to work with FSX.

You could always try it out as is Both FSX and MSFS2020 use SimConnect, and that's what the Flight Tracker plugin talks to. Theoretically, it should just work, since the SimConnect commands between both sims are the same and they communicate via the same interface.

I could be horribly wrong about this, but it won't hurt to try.

**Reply by nguyenquyhy** [reply]
It might not work out of the box as FSX is 32-bit and MSFS is upgraded to 64-bit. This means you might get some error loading SimConnect library.

However, when I look around the SDK samples, I see something about turning on compatibility mode with FSX in a config file. I haven't really look into it yet, but I think they try to leverage SimConnect's ability to work over network for that.

**Reply by Millsen** [reply]
Hey guys, I could need some help here.
I want to use FlightTracker for FSX but I can't get it to work.
I have in FSX 4 simconnect installed (RTM, SP1 XPACK and 10.062615.0)
Are there any steps I have to do after installing the plugin? Unfortunately I have not understand your latest reply with the "compatibility mode with FSX in a config file" I am not a programmer, so please try to explain step by step that I know where to go, and where I have to do what.

THANK YOU VERY MUCH IN ADVANCE

**Reply by awralls** [reply]
Can I add my voice to a plea for guidance on getting this to work with FSX? Seems like a potentially super idea.

Cheers,

Andy

**Reply by Costa1999** [reply]
me too!
